### PR TITLE
Fix link to worldengine-gui project

### PR DESCRIPTION
**Pull request by mrcasals**
This PR fixes the link to the GUI project on the README file.

### CODE REVIEW

**Comment by ftomassetti** [comment]
Thanks for fixing this. It is really appreciated!

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ For Windows, Linux and Mac are available on the [releases page](https://github.c
 Gui
 ===
 
-An experimental (and limited!) GUI is available as a separate project: [https://github.com/Mindwerks/worldengine-gui](worldengine-gui).
+An experimental (and limited!) GUI is available as a separate project: [https://github.com/Mindwerks/worldengine-gui](https://github.com/Mindwerks/worldengine-gui).
```
Comment (tcld) [inline README.md]: You should change the name (the stuff inside ```[]```) to "worldengine-gui" now.

Comment (ftomassetti) [inline README.md]: I thought about that but it can also sense to show the entire URL. What is critical is to have a working link

**Comment by tcld** [comment]
Mmh...sorry for being five seconds too late.^^ I guess the name will have to be fixed as a separate PR, then.

File: README.md (hunk 1 above)
Comment (tcld) [inline README.md]: Sure. But other links in the README are masked like that (which might not be the correct word) so... well, it might not matter.

Comment (ftomassetti) [inline README.md]: right, it could make sense to adapt the name of the link then

**Pull request merged**